templates/common/.devcontainer/devcontainer.json/java/devcontainer.json

```diff
@@ -7,11 +7,19 @@
         }
     },
     "features": {
-        "github-cli": "2",
-        "azure-cli": "2.38",
-        "docker-from-docker": "20.10",
-        "java": "17.0",
-        "node": {
+        "ghcr.io/devcontainers/features/github-cli:1": {
```

Comment:
I would be interested to learn some more about this syntax.  Would you mind sharing a link about it?
Wondering about the `:1` at the end...

Reply:
Sure, you can learn more here:
https://github.com/devcontainers/features
The 1 at the end is the version of the feature, versus the version of the language/package inside it, I believe. I only see 1 as an option here:
https://github.com/devcontainers/features/tree/main/src/github-cli